graphix/command.py

```diff
@@ -5,8 +5,10 @@
 import abc
 import enum
 
+import numpy as np
 from pydantic import BaseModel
 
+import graphix.clifford
```

Comment:
`from graphix.clifford import Clifford` ?

Reply:
`Clifford` used later

Reply:
Fixed in f48ee93.